_source/logzio_collections/_log-sources/kubernetes.md

```diff
@@ -65,52 +73,101 @@ The logzio-k8s image comes pre-configured for Fluentd to gather all logs from th
     see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
 {:.tasklist.firstline-headline}
 
-## Disabling systemd input
+</div>
+<!-- tab:end -->
 
-To suppress Fluentd system messages, set the `FLUENTD_SYSTEMD_CONF` environment variable to `disable` in your Kubernetes environment.
 
-<!-- ## Customizing the configuration {#customizing-the-configuration}
+<!-- tab:start -->
+<div id="custom-config">
+
+## Deploy logzio-k8s with custom configuration
+
+You can customize the configuration of the Fluentd container.
+This is done using a ConfigMap that overwrites the default DaemonSet.
+
+###### To deploy logzio-k8s
+
+1.  Store your Logz.io credentials
+
+    Save your Logz.io shipping credentials as a Kubernetes secret.
+
+    {% include log-shipping/replace-vars.html token=true listener=true %}
+
+    ```shell
+    kubectl create secret generic logzio-logs-secret \
+      --from-literal=logzio-log-shipping-token='<<ACCOUNT-TOKEN>>' \
+      --from-literal=logzio-log-listener='https://<<LISTENER-HOST>>:8071' \
+      -n kube-system
+    ```
+
+2.  Download the ConfigMap
+
+    Download `k8s-fluentd-custom.yaml` from [https://raw.githubusercontent.com/logzio/logzio-k8s/master/k8s-fluentd-custom.yaml]()
+    and open the file in your text editor.
```

Comment:
The link is not working

Reply:
That's because it doesn't exist yet. We decided to add another file for a custom config with a configMap addition. I go this..

Reply:
Cool